src/Akeneo/Pim/Structure/Bundle/Infrastructure/Job/JobParameters/ConstraintCollectionProvider/MoveAttributeMassEdit.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @copyright 2023 Akeneo SAS (https://www.akeneo.com)
+ * @license   https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Akeneo\Pim\Structure\Bundle\Infrastructure\Job\JobParameters\ConstraintCollectionProvider;
+
+use Akeneo\Tool\Component\Batch\Job\JobInterface;
+use Akeneo\Tool\Component\Batch\Job\JobParameters\ConstraintCollectionProviderInterface;
+use Symfony\Component\Validator\Constraints\All;
+use Symfony\Component\Validator\Constraints\Collection;
+use Symfony\Component\Validator\Constraints\NotNull;
+use Symfony\Component\Validator\Constraints\Type;
+
+final class MoveAttributeMassEdit implements ConstraintCollectionProviderInterface
```

Comment:
IMO, we should name this class DeleteAttributeGroups because it's the job name and it's his purpose, moving attributes is a side effect
```suggestion
final class DeleteAttributeGroupsMassEdit implements ConstraintCollectionProviderInterface
```